Fix release process and package publish (#1203)
ably/ably-dotnet: CONTRIBUTING.md

  - Also ensure that the "Full Changelog" link points to the new version tag instead of the `HEAD`.
5. Commit this change: `git add CHANGELOG.md && git commit -m "Update change log."`.
6. Push the branch and create a release PR (ensure you include an SDK Team Engineering Lead and the SDK Team Product Manager as reviewers) and gain approvals for it, then merge that to `main`.

> 7. Go to [Github Actions tab](https://github.com/ably/ably-dotnet/actions), click on [Package Ably](https://github.com/ably/ably-dotnet/actions/workflows/package.yml) workflow at the left nav-bar. On the right corner, click on `Run workflow` `with 1.2.3` (current release version) as a input to `Ably version`.

Reviewer: ```suggestion
7. Go to [Github Actions tab](https://github.com/ably/ably-dotnet/actions), click on [Package Ably](https://github.com/ably/ably-dotnet/actions/workflows/package.yml) workflow at the left nav-bar. On the right corner, click on `Run workflow` with the current release tag as the input to `Ably version`.
```

>   - You can check all latest workflows under [Github Actions Tab](https://github.com/ably/ably-dotnet/actions). Donwload the generated artifact named `save-nuget-package` at the end of the latest successful workflow run.

Reviewer: ```suggestion
  - You can check all latest workflows under [Github Actions Tab](https://github.com/ably/ably-dotnet/actions). Download the generated artifact named `save-nuget-package` at the end of the latest successful workflow run.
```
